### ISSUE TITLE
Summernote is not showing when Model Value is null/empty

### ISSUE THREAD

**Issue by bastisk** [martinsvb/ng2-summernote#8]
Hi there,
I noticed that when the model which is assigned to the <ng2-summernote> directive is null or even an empty string like "" - the summernote will not be rendered. I think this behavior does not make sense as the text can be empty (e.g. if summernote is used in an input form).

Let me know if you need any more details on this to understand  the problem.

**Comment by sascha1337** [comment]
Same here - please fix !

**Comment by BrianWClausen** [comment]
I am having the same issue. Everything else seems to be working really well.

**Comment by abhinashtripathy** [comment]
Did anyone find any solution or work around to this issue??

**Comment by segvan** [comment]
yes there is a bug in writeValue(value: any) method. Instead of  if (value) { ... } should be if (value !== 'undefined' && value !== null) { ... }
